torch/_tensor_str.py

```diff
@@ -70,7 +71,8 @@ def set_printoptions(
 
 class _Formatter(object):
     def __init__(self, tensor):
-        self.floating_dtype = tensor.dtype.is_floating_point
+        self.floating_dtype = tensor.is_floating_point()
+        self.complex_dtype = tensor.is_complex
```

Comment:
Could you please comment on the reason of difference between `is_floating_point()` (function call) and `is_complex` (attribute/property access)

Reply:
> Could you please comment on the reason of difference between `is_floating_point()` (function call) and `is_complex` (attribute/property access)

No complex integers are not supported yet. That makes sense, however I read this in one of the numpy user guides:
“There are 5 basic numerical types representing booleans (bool), integers (int), unsigned integers (uint) floating point (float) and complex. ” https://docs.scipy.org/doc/numpy-1.3.x/numpy-user.pdf and so I thought we should not interpret complex numbers as floating point.

Reply:
I agree it should be consistent. fixed